arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5515,7 +5658,8 @@ void ClusterInfo::loadServers() {
   auto [acb, index] = agencyCache.read(
       std::vector<std::string>({AgencyCommHelper::path(prefixServersRegistered),
                                 AgencyCommHelper::path(mapUniqueToShortId),
-                                AgencyCommHelper::path(prefixServersKnown)}));
+                                AgencyCommHelper::path(prefixServersKnown),
+                                AgencyCommHelper::path(prefixHealth)}));
```

Comment:
`AgencyCommHelper::path(prefixHealth)`
Can it slowdown loadServers?

Reply:
We was anyway triggered if this key changes ( see HeartbeatThread). So now we just read the key. So I don't expect significant difference.